docs/adr/0012-high-level-design-infra-cluster.md

```diff
@@ -0,0 +1,119 @@
+## ABOUT THIS DOCUMENT
+
+This document is a High-level Design for the new MOC infra management cluster. This hub cluster will be the central openshift cluster to manage other clusters on-demand using infrastructure as a code to deploy configuration. This centralized cluster will be part of Operate First with RedHat products, testing new dev preview capabilities and providing feedback.
+
+This document will be publicly shared for the open-source community for others to learn. Providing a full technical stack overview that can be automated and step up in one-click operation.
+
+
+## AUDIENCE
+
+The intended audience for this document is any person with interest in deploying a similar solution as described in this document.
+
+## REVISIONS
+
+
+| Name | Version | Date | Comments |
+|------| --------| -----| ---------|
+| Ilana Polonsky | 1.0 | 02/16/2021 | MOC infra leader |
+| Oren Oichman | 1.1 | 02/16/2021 | Senior Cloud Consultant |
+
+## CURRENT STATUS
+
+Our current status consisted of three separate OpenShift clusters one for running development tasks, one for Infrastructure testing and a single (all in one) node cluster.
+
+### Details about the current clusters
+
+*   Lars’s cluster - A 6 nodes openshift cluster that runs Argocd to deploy application and configuration.
+*   Ilana’s cluster - a 4 node BM cluster that serves for the ‘CNV on MOC’ initiative and was installed using the IPI installer.
+*   All in one cluster - single node cluster running on a VM, with ACM as a managing component for Ilana's cluster.
+*   Currently all of our infrastructure environment is consistent with Bare metal Servers (the All in One cluster is running on a VM in the provisioner server.
+*   The Environment has a constant connection to the internet.
+
+The current system offers users a non-production environment to run Kubernetes workload, all the clusters are not prominent and are changed/modified weekly.
+
+
+## BASIC ASSUMPTIONS
+
+
+*   In this solution we are going to use only products that are provided by Red Hat or will be provided by Red Hat in the near future.
+*   We are using a BM only environment which resides in MOC (mass open cloud) Datacenter.
+*   All our deployment steps will be documented.
+
+
+## TOOLS
+
+There are two tools to accomplish our desire infrastructure:
+
+*   Using the cloud.redhat.com tool to deploy the cluster
+*   Using the Openshift install binary (KNI) to deploy the cluster (IPI).
+
+### Pros and Cons for cloud.redhat.com (Assisted Installer):
+**Pros:**
+
+- Assisted Installer is a growing RedHat tool to install a new cluster, getting family with it will have future benefits.
+- This tool will be part of the ACM and will be used in deploying new clusters via the ACM.
+- It's a good opportunity to provided feedback to the developers.
+- Can be easily automated, use API calls.
+
+**Cons:**
+- A dev preview tool, not completed product
+- This method includes booting from an image a BM, a more complicated way.
```

Comment:
I'm not entirely sure what you're trying to say here. Maybe:

> This method requires booting baremetal nodes from an ISO image

I'm not sure this is actually a "con", arguably, booting from virtual media is easier than setting up the PXE server required by the baremetal installer.